coalib/results/result_actions/ShowPatchAction.py

```diff
@@ -67,8 +67,9 @@ class ShowPatchAction(ResultAction):
     @staticmethod
     @enforce_signature
     def is_applicable(result: Result, original_file_dict, file_diff_dict):
+
         if not result.diffs:
-            return False
+            return 'This result cannot be automatically fixed.'
```

Comment:
how about 'This result has no patch attached' only as ShowPatchAction doesnt fix the code